src/config/dynamic-theme-fixes.config

```diff
@@ -41,6 +41,19 @@ div[class^='scrollableList'] {
 
 ================================
 
+*.itslearning.com
+
+INVERT
+.ccl-feedback-tip
+.c-dropdownMenu__navigation
```

Comment:
Yes it's behind an auth wall. I can attach an screenshot if you like (but I'd have to censor some parts because there are some private information there)

Reply:
Seems good for now.